app/components/explore/category-sidebar/general-categories.js

```diff
@@ -12,6 +13,7 @@ export default Component.extend({
   init() {
     this._super(...arguments);
     get(this, 'getCategoriesTask').perform();
+    this.sticky = new Sticky('.is-sticky');
```

Comment:
I feel like these should be done in the `didInsertComponent` hook, rather than initialization of the Component module.